iotilecore/iotile/core/utilities/hash_algorithms/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""Hashing algorithms that we commonly use."""
+
+from collections import namedtuple
+from .crc import calculate_crc
+from .sha import calculate_sha
+
+HashAlgorithm = namedtuple("HashAlgorithm", ['calculate', 'algorithm'])
+
+KNOWN_HASH_ALGORITHMS = {
+    'crc32_0x104C11DB7': HashAlgorithm(calculate_crc, 0x104C11DB7),
+    'sha256': HashAlgorithm(calculate_sha, 256)
+}
+
+__all__ = ['KNOWN_HASH_ALGORITHMS']
```

Comment:
This `__init__.py` file is where we want to list frequently used and known hashing algorithms. This package will allow a programmer to easily select the proper hashing functions based on the index they provide to the dict.

For example, they can select the sha256 algorithm by doing this
```
hash_alg = KNOWN_HASH_ALGORITHMS['sha256']
hash_alg.calculate(hash_alg.algorithm, some_bytes)
```